test/hotel-test.js

```diff
@@ -0,0 +1,108 @@
+import { expect } from 'chai';
+import Hotel from '../src/classes/Hotel';
+import Room from '../src/classes/Room';
+import Booking from '../src/classes/Booking';
+import sampleCustomers from '../src/data/customers-data';
+import sampleBookings from '../src/data/bookings-data';
+import sampleRooms from '../src/data/rooms-data';
+
+describe('Hotel', () => {
+  let hotel;
+  let allRooms;
+  let allBookings;
+
+  beforeEach(() => {
+    hotel = new Hotel(sampleRooms, sampleBookings);
+    allRooms = sampleRooms.map(room => new Room (room));
+    allBookings = sampleBookings.map(booking => new Booking(booking));
+  });
+
+  it('should be a function', () => {
+    expect(Hotel).to.be.a('function');
+  });
+
+  it('should be an instance of Hotel', () => {
+    expect(hotel).to.be.an.instanceof(Hotel);
+  });
+
+  it('should have a property that stores all rooms', () => {
+    expect(hotel.rooms).to.be.an('array');
+    expect(hotel.rooms.length).to.be.greaterThan(0);
+    expect(hotel.rooms).to.deep.equal(allRooms);
+  });
+
+  it('should store all rooms as instances of Room', () => {
+    expect(hotel.rooms[0]).to.be.an.instanceof(Room);
+  });
+
+  it('should have a property that stores all bookings', () => {
+    expect(hotel.bookings).to.be.an('array');
+    expect(hotel.bookings.length).to.be.greaterThan(0);
+    expect(hotel.bookings).to.deep.equal(allBookings);
+  });
+
+  it('should store all bookings as instances of Booking', () => {
+    expect(hotel.bookings[0]).to.be.an.instanceof(Booking);
+  });
+
+  it('should show all rooms available for a given date', () => {
```

Comment:
Could this a case where you test that something unique happens if there are NO rooms available? I think I'd probably handle it in my dom manipulation which is hard to test. But I figured I'd share the thought.

Reply:
That's a great point! I'm going to add in a test for this method with a date that should return no rooms available.